test/helpers/tokens.js

```diff
@@ -0,0 +1,47 @@
+
+const ERC20TokenList = async (params, from, tokenFactory) => {
```

Comment:
naming suggestion `getErc20TokenInstances`

Reply:
afaiu the `tokenFactory` param is not used

Reply:
also if root is always the deployer here (haven't checked if this is the case), then you could just import root within this function `const {root} = await ethers.getNamedSigners()`. If this is the case I would leave the `from` parameter.

Reply:
I think it is beneficial to keep the `from` parameter because the `getErc20TokenInstances` will be more general in that way, which would benefit testing overall.